endorse/commit.go

```diff
@@ -267,10 +272,12 @@ func snapshotEndorsement(ctx context.Context, cops ChangeOps, endorsement *epb.V
 		return err
 	}
 	fwPath := ec.VCS.ReleasePath(ctx, path.Join(ec.SnapshotDir, ec.ImageName))
+	fwSigPath := fmt.Sprintf("%s.signed", fwPath)
+	fwEvtsPath := fmt.Sprintf("%s.evts.pb", fwPath)
 	svsmPath := ec.VCS.ReleasePath(ctx, path.Join(ec.SnapshotDir, "svsm.igvm"))
-	sigPath := fmt.Sprintf("%s.signed", fwPath)
-	evtsPath := fmt.Sprintf("%s.evts.pb", fwPath)
-	if err := writeEndorsement(ctx, sigPath, endorsement, cops); err != nil {
+	svsmSigPath := fmt.Sprintf("%s.signed", svsmPath)
+	svsmEvtsPath := fmt.Sprintf("%s.evts.pb", svsmPath)
+	if err := writeEndorsement(ctx, []string{fwSigPath, svsmSigPath}, endorsement, cops); err != nil {
```

Comment:
Do we only want to write the svsm.igvm.signed and svsm.igvm.events.pb file if SvsmImage is set?

Reply:
Done.